packages/vm/lib/runTx.ts

```diff
@@ -153,8 +153,13 @@ export default async function runTx(this: VM, opts: RunTxOpts): Promise<RunTxRes
     await state.commit()
     debug(`tx checkpoint committed`)
     if (this._common.isActivatedEIP(2929) && opts.reportAccessList) {
+      const { tx } = opts
+      // Do not include sender address in access list
+      const removed = [tx.getSenderAddress()]
+      // Only include to address on present storage slot accesses
+      const onlyStorage = tx.to ? [tx.to] : []
       // @ts-ignore method is not yet part of the interface for backwards-compatibility reasons
```

Comment:
I think this will lead to some very uncomfortable patterns. For example, we use `StateManager` everywhere, as we have two different implementations.

What would be better is making this new method optional. That requires less type casting, but rather checking `!== undefined` before using it.

Reply:
How do I make a method optional in an interface? 🤔

Reply:
Ah, ok, got it https://stackoverflow.com/questions/27522973/typescript-optional-function-in-interface, I somehow was convinced that this is not possible, will update.